utils/frame/frame-utilities-cli/src/transfer.rs

```diff
@@ -0,0 +1,155 @@
+    AddressFor, IndexFor, AccountIdFor,
+    SignedExtensionProvider, CallFor,
+};
+use pallet_balances::Call as BalancesCall;
+use crate::utils::create_extrinsic_for;
+
+type Bytes = Vec<u8>;
+
+/// Balance type
+type BalanceFor<R> = <R as pallet_balances::Trait>::Balance;
+
+/// The `transfer` command
+#[derive(Debug, StructOpt)]
+#[structopt(
+    name = "transfer",
+    about = "Author and sign a Node pallet_balances::Transfer transaction with a given (secret) key"
+)]
+pub struct TransferCmd {
+    /// The number of units to transfer.
+    #[structopt(long)]
+    amount: GenericNumber,
+
+    /// The signing secret key URI.
+    #[structopt(long)]
+    from: String,
+
+    /// The signing account's transaction index.
+    #[structopt(long)]
+    index: GenericNumber,
+
+    /// The destination account public key URI.
+    #[structopt(long)]
+    to: String,
+
+    /// genesis hash, for signed extensions.
+    #[structopt(long, parse(try_from_str = decode_hex))]
+    prior_block_hash: Bytes,
+
+    #[allow(missing_docs)]
+    #[structopt(flatten)]
+    keystore_params: KeystoreParams,
+
+    #[allow(missing_docs)]
+    #[structopt(flatten)]
+    shared_params: SharedParams,
+
+    #[allow(missing_docs)]
+    #[structopt(flatten)]
+    crypto_scheme: CryptoSchemeFlag,
+}
+
+
+impl TransferCmd {
+    /// Run the command
+    pub fn run<R>(&self) -> Result<(), Error>
+        where
+            R: pallet_balances::Trait + pallet_indices::Trait + SignedExtensionProvider,
+            AccountIdFor<R>: for<'a> TryFrom<&'a [u8], Error = ()> + Ss58Codec + From<AccountId32>,
+            AddressFor<R>: From<AccountIdFor<R>>,
+            <IndexFor<R> as FromStr>::Err: Debug,
+            <BalanceFor<R> as FromStr>::Err: Debug,
+            CallFor<R>: Encode + From<BalancesCall<R>>,
+            BalancesCall<R>: Encode,
+    {
+        let password = self.keystore_params.read_password()?;
+        let nonce = self.index.parse::<IndexFor<R>>()?;
+        let to = if let Ok(data_vec) = decode_hex(&self.to) {
+            AccountIdFor::<R>::try_from(&data_vec)
+                .map_err(|_| "Invalid hex length for account ID; should be 32 bytes")?
+        } else {
+            AccountIdFor::<R>::from_ss58check(&self.to)
+                .map_err(|_| "Invalid SS58-check address given for account ID.")?
+        };
+        let amount = self.amount.parse::<BalanceFor<R>>()?;
+        let prior_block_hash = <R::Hash as Decode>::decode(&mut &self.prior_block_hash[..])?;
+
+        with_crypto_scheme!(
+			self.crypto_scheme.scheme,
+			print_ext<R>(
+				&self.from,
+				password.as_ref().map(String::as_str),
+				to.into(),
+				nonce,
+				amount,
+				prior_block_hash
+			)
+		)
+    }
+}
+
+impl CliConfiguration for TransferCmd {
+    fn shared_params(&self) -> &SharedParams {
+        &self.shared_params
+    }
+
+    fn keystore_params(&self) -> Option<&KeystoreParams> {
+        Some(&self.keystore_params)
+    }
+}
+
+fn print_ext<Pair, P>(
```

Comment:
Function already exists in the other file

Reply:
yeah they should be unified